src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -570,6 +575,15 @@ internal bool CheckIfAttributeShouldBeEmbedded(EmbeddableAttributes attribute, D
                         WellKnownType.System_Runtime_CompilerServices_NullablePublicOnlyAttribute,
                         WellKnownMember.System_Runtime_CompilerServices_NullablePublicOnlyAttribute__ctor);
 
+                case EmbeddableAttributes.NativeIntegerAttribute:
```

Comment:
I'm surprised that DynamicAttribute is not in some of these code paths. Is it something we never synthesize?

Reply:
`System.Runtime.CompilerServices.DynamicAttribute` is included in the framework.

---
In reply to: [373266813](https://github.com/dotnet/roslyn/pull/41160#discussion_r373266813) [](ancestors = 373266813)